style.css

```diff
@@ -0,0 +1,128 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html,
+body {
+  font-size: 18px;
+  font-family: "Poppins", sans-serif;
+  width: 100%;
+  overflow-x: hidden;
+}
+
+.navbar {
+  background-color: white;
+}
+
+.navbar-content {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-left: 40px;
+  margin-right: 40px;
+  margin-top: 40px;
+  border: none;
+}
+
+.navbar-content img {
+  width: 20px;
+  height: 13px;
+}
+
+.logo {
+  color: #6070ff;
+  font-weight: 700;
+  text-decoration: none;
+}
+
+.nav_link ul {
+  display: none;
+}
+
+.section-1 {
+  height: 720px;
+  background-color: #f7f7f9;
+  border-style: none;
+}
+
+.section-1-container {
+  background-image: url("images/header-shapes\ mobile.png");
+  background-repeat: no-repeat;
+  background-position: center;
+  border-bottom-left-radius: 100px;
+  background-color: white;
+  padding: 160px 25px;
+  width: 100%;
+  height: 100%;
+}
+
+.heading {
+  font-size: 2.5em;
+  font-weight: 700;
+  line-height: 52px;
+}
```

Comment:
- [OPTIONAL] Kindly make sure your main title has the same color as the one on the provided Figma design. The used color is `#172B4D`, add it to your heading it will be identical and match the provided Figma design.

   - ### YOUR DESIGN
    
   ![image](https://user-images.githubusercontent.com/87197412/234823767-9c01f078-970c-4553-be10-223031834415.png)

   - ### FIGMA DESIGN
   
   ![image](https://user-images.githubusercontent.com/87197412/234823634-72276a94-684f-4ef1-bb94-e8ee6765545e.png)